lib/travis/api/v3/queries/log.rb

```diff
@@ -18,7 +18,7 @@ def find(job)
     end
 
     def create_log_parts(log, content)
-      log.log_parts << Models::LogPart.new(log_id: log.id, content: content, number: 0, created_at: log.created_at)
+      log.log_parts.build([{content: content, number: 0, created_at: log.created_at}])
```

Comment:
this was required because otherwise the `<<` operator was actually creating a real log_part on the database, which isn't what we wanted. We only want a pretend log_part...